graylog2-web-interface/src/util/FormsUtils.js

```diff
@@ -14,6 +14,11 @@ const FormUtils = {
         return input.value;
     }
   },
+
+  // Emulates a minimal input event that can be used with FormUtils#getValueFromInput()
+  inputEvent(name, value) {
```

Comment:
This does not seems to be used at the moment, and I'm also not sure what is the reasoning for adding this code. If we need a fake event for some reason, I guess we should use at least an instance of [`CustomEvent`](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent).